runtime/jvmti/jvmtiThread.c

```diff
@@ -526,7 +526,10 @@ jvmtiGetThreadInfo(jvmtiEnv *env,
 					}
 				} else {
 					/* For platform threads, a NULL vmthread indicates that a thread is terminated. */
```

Comment:
please also update the comments here, only terminated threads don't set the threadGroup

Reply:
```suggestion
					/* For platform threads, a NULL vmthread indicates that a thread is either unstarted or terminated. */
```

Reply:
@fengxue-IS @babsingh added comments